app/forms/declaration/AdditionalInformation.scala

```diff
@@ -41,7 +41,7 @@ object AdditionalInformation extends DeclarationPage {
         .verifying("declaration.additionalInformation.description.empty", nonEmpty)
         .verifying(
           "declaration.additionalInformation.description.error",
-          isEmpty or (noLongerThan(70) and isAlphanumericWithAllowedSpecialCharacters)
+          isEmpty or (noLongerThan(70) and isAlphanumericWithSpecialCharacters(Set('\r', '\n') ++ allowedSpecialChars))
```

Comment:
Could the method `isAlphanumericWithAllowedSpecialCharacters` stay? To avoid exposing private field from `FieldValidator`.

Reply:
Done